react/tic-tac-toe/src/utils/Validations/index.js

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line no-useless-escape
+const VALID_EMAIL = /^\w+([\.-]?\w+)*@wolox.com/;
+
+const isRequired = value => value ? undefined : 'Required field';
+
+const hasLength = value =>
+  value.length < 8 ? 'Value length has to be greater than 8 characters' : undefined;
+
+const isValidEmail = value =>
+  VALID_EMAIL.test(value) ? undefined : 'Domain needs to be @wolox.com';
+
+export const validateEmail = value => [
+  isRequired(value),
+  isValidEmail(value)
+];
+
+export const validatePassword = value => [
```

Comment:
I don't think this will work, your field is expecting to receive an array with all your validation functions. 

```
export const validatePassword = [
  isRequired, 
  hasLength
];